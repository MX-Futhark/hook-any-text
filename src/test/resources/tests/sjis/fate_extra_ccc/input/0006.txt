82DE814282A282E2814196B3905F8C6F82C88352836782F08CFB82C982B582BD81420A90B6936B89EF82C882C796B3979D8BAD82A282B582C482DC82C50A93FC82C182C482E082E782A482E082CC82C582CD82C882A982C182BD82A98142008200000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C0000001D0000001E0000001F000000200000002100000022000000230000002400000025000000260000002700000028000000290000002A0000002B0000002C0000002D00000B000000F014C90800000000000000000000000000006D00000000000000504300000000FFFFFFFF0000000000000000FFFFFFFF0000000000008843FFFFFFFF0000504300000000FFFFFFFF0000504300008843FFFFFFFF00000000000088434C15C9080000000000000000FFFFFFFF000050430000884300005043000000000B0000007015C90800000000000000000000000001006D00000000000000504300000000FFFFFFFF0000000000000000FFFFFFFF0000000000008843FFFFFFFF0000504300000000FFFFFFFF0000504300008843FFFFFFFF0000000000008843000000000000000000000000FFFFFFFF00005043000088434CA5A3480B00000002000000000000000000000000000000000000000600C20100000000D47EA3480B0000000000000000000000000000000000000002001000000000000000000000000000FFFFFFDC0000000000003C43000000000000F04300000000FFFFFFDC0000F04300003C4300000000000000000000A842FFFFFFDC0000000000008843000000000000F0430000A842FFFFFFDC0000F0430000884300000000000000000B0000000000000000000000000000000000000003001000