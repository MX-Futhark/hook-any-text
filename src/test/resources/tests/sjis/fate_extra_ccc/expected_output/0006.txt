["む。いや、無神経なコトを口にした。
生徒会など無理強いしてまで
入ってもらうものではなかったか。"]